jck/diff/diff.sh

```diff
@@ -8,18 +8,21 @@ setup()
 	if [[ $VERSION1 == $VERSION2 ]] ; then 
 		echo "Please provide two different repositories to compare"
 		exit 1; 
-	fi 
-
+	fi
+
+	if [[ "$VERSION1" > "$VERSION2" ]]; then
+		temp="$VERSION1"
+		VERSION1="$VERSION2"
+		VERSION2="$temp"
+		temp="$REPO1"
+		REPO1="$REPO2"
+		REPO2="$temp"
+	fi
+
+	VERSION_VALUE1="$VERSION1"
+	VERSION_VALUE2="$VERSION2"
 	if [[ "$VERSION1" -eq 8 ]] ; then 
```

Comment:
I think we should use 8c,  as the input param, rather than coding in this script, eg 11 has a suffix as well (11a)